README.md

```diff
@@ -197,10 +212,9 @@ interface IInjectedProvidersMap {
     - key: the secret key (required)
     - network: choose initial network name (optional)
 
-You can disable the injected provider or WalletConnect by adding the following keys:
+You can disable the injected provider by adding the following keys:
 
 - disableInjectedProvider: true (optional)
-- disableWalletConnect: true (optional)
```

Comment:
The new WalletConnect web3-provider release introduced a required parameter (infuraId) which made the disableWalletConnect flag redundant since you can simply not pass its specific options